h2o-hadoop-2/h2o-mapreduce-2-generic/build.gradle

```diff
@@ -1,6 +1,7 @@
 ext {
   hadoopVersion = 'cdh5.2'
```

Comment:
I have no idea why we have CDH 5.2 here. Seems incorrect to me.

Reply:
agree, but thats for another PR